README.md

```diff
@@ -58,7 +59,8 @@ fn main() {
     println!("Are the tokens equal? {}", dai_token.equals(&another_dai_token));
 
     // Example of sorting tokens
-    let another_token = token!(CHAIN_ID, "0x0000000000000000000000000000000000000002", DECIMALS, "ETH", "Ethereum");
+    const ANOTHER_TOKEN_ADDRESS: &str = "0x0000000000000000000000000000000000000002";
+    let another_token = token!(CHAIN_ID, ANOTHER_TOKEN_ADDRESS, DECIMALS, "ETH", "Ethereum");
```

Comment:
```suggestion
    let another_token = token!(
        CHAIN_ID,
        "0000000000000000000000000000000000000002",
        DECIMALS,
        "ETH",
        "Ethereum"
    );
```
`token!` can take either string literal or `&str`. If passing string literal to `address!` macro, "0x" should be stripped.